src/components/ExecutorDetails.js

```diff
@@ -0,0 +1,311 @@
+
+                            } }
+                        /> }
+                    </CardBody>
+                </Card>
+            </Stack>
+        </Main>,
+        success: <Main>
+            <Stack gutter="md">
+                <Card>
+                    <CardHeader className='ins-m-card__header-bold'>
+                        <Button
+                            variant='link' onClick={ () => downloadPlaybook(remediation.id) }>
+                            Download Playbook
+                        </Button>
+                    </CardHeader>
+
+                    <CardBody>
+                        { InventoryTable && <InventoryTable
+                            ref={ inventory }
+                            items={ orderBy(systems, [ s => getSystemName(s), s => s.id ]) }
+                            onRefresh={ onRefresh }
+                            page={ page }
+                            total={ systems.length }
+                            perPage={ pageSize }
+                            tableProps={ { onSelect: undefined } }
+                            expandable
+                            onExpandClick={ (_e, _i, isOpen, { id }) => {
+                                getPlaybookRunSystemDetails(remediation.id, run_id, id);
+                                onCollapseInventory(isOpen, id);
+                            } }
+                        /> }
+                    </CardBody>
+                </Card>
+            </Stack>
+        </Main>,
+        failure: <Main>
+            <Stack gutter="md">
+                <Card>
+                    <CardHeader className='ins-m-card__header-bold'>
+                        <Button
+                            variant='link' onClick={ () => downloadPlaybook(remediation.id) }>
+                            Download Playbook
+                        </Button>
+                    </CardHeader>
+
+                    <CardBody>
+                        <RunFailed name={ executor.executor_name }/>
+                    </CardBody>
+                </Card>
+            </Stack>
+        </Main>
+    })[normalizeStatus(status)];
+
+    const permission = useContext(PermissionContext);
+
+    return remediation && executor && playbookRun && playbookRun.data
+        ? <React.Fragment>
+            <PageHeader>
+                <Breadcrumb>
+                    <BreadcrumbItem>
+                        <Link to={ `/${remediation.id}` }> { remediation.name } </Link>
+                    </BreadcrumbItem>
+                    <BreadcrumbItem>
+                        <Link to={ `/${remediation.id}/${run_id}` }>  <DateFormat type='exact' date={ playbookRun.data.created_at } /> </Link>
+                    </BreadcrumbItem>
+                    <BreadcrumbItem isActive> { executor.executor_name } </BreadcrumbItem>
+                </Breadcrumb>
+                <Stack gutter>
+                    <StackItem>
+                        <PageHeaderTitle title={ executor.executor_name } />
```

Comment:
When running there should be the running icon (see https://marvelapp.com/j3b9b7e/screen/66225551)